src/components/learner-credit-management/BudgetAssignmentsTable.jsx

```diff
@@ -1,23 +1,41 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { DataTable } from '@edx/paragon';
-
+import {
+  Button,
+  DataTable,
+} from '@edx/paragon';
 import TableTextFilter from './TableTextFilter';
 import CustomDataTableEmptyState from './CustomDataTableEmptyState';
 import AssignmentDetailsTableCell from './AssignmentDetailsTableCell';
 import AssignmentStatusTableCell from './AssignmentStatusTableCell';
+import AssignmentRowActionTableCell from './AssignmentRowActionTableCell';
 import { DEFAULT_PAGE, PAGE_SIZE, formatPrice } from './data';
 import AssignmentRecentActionTableCell from './AssignmentRecentActionTableCell';
 
 const FilterStatus = (rest) => <DataTable.FilterStatus showFilteredFields={false} {...rest} />;
 
+const CancelAction = ({ selectedFlatRows, ...rest }) => (
+  // eslint-disable-next-line no-console
+  <Button variant="danger" onClick={() => console.log('Cancel', selectedFlatRows, rest)}>
+    {`Cancel (${selectedFlatRows.length})`}
+  </Button>
+);
+
+const RemindAction = ({ selectedFlatRows, ...rest }) => (
+  // eslint-disable-next-line no-console
+  <Button onClick={() => console.log('Remind', selectedFlatRows, rest)}>
+    {`Remind (${selectedFlatRows.length})`}
+  </Button>
+);
```

Comment:
[suggestion] Might recommend creating these actions in standalone files, one per component, as I imagine these components will grow in size once they integrate with opening modals, etc.

Reply:
ahh yeah perhaps- it was just so small I didn't think necessary but I agree